barretenberg/cpp/src/barretenberg/srs/global_crs.hpp

```diff
@@ -12,7 +12,9 @@ void init_grumpkin_crs_factory(std::string crs_path);
 void init_grumpkin_crs_factory(std::vector<curve::Grumpkin::AffineElement> const& points);
 void init_crs_factory(std::vector<bb::g1::affine_element> const& points, bb::g2::affine_element const g2_point);
 
-std::shared_ptr<bb::srs::factories::CrsFactory<curve::BN254>> get_crs_factory();
-std::shared_ptr<bb::srs::factories::CrsFactory<curve::Grumpkin>> get_grumpkin_crs_factory();
+std::shared_ptr<factories::CrsFactory<curve::BN254>> get_bn254_crs_factory();
```

Comment:
Added these in prep for simplifying crs factory interface but haven't used the uniform function yet.